frontend/public/locales/en-GB/onboarding.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Create a task to get rid of en-GB files as most of the content looks same.

Reply:
https://github.com/SigNoz/signoz/issues/5446